### ISSUE TITLE
Unable to build Clasp. Duplicated Lines of Code in file auth.ts

### ISSUE THREAD

**Issue by valen22br** [google/clasp#297]
(Please search [existing issues](https://github.com/google/clasp/issues) before creating a new one.)

## Expected Behavior
Have clasp build after run the command npm run build

## Actual Behavior
error during the building process:
> tsc --project tsconfig.json && npm i -g --loglevel=error

src/auth.ts:1:13 - error TS2300: Duplicate identifier 'fs'.

1 import * as fs from 'fs';
              ~~

src/auth.ts:2:13 - error TS2300: Duplicate identifier 'fs'.

2 import * as fs from 'fs';
              ~~

src/auth.ts:11:10 - error TS2300: Duplicate identifier 'Discovery'.

11 import { Discovery } from 'googleapis/build/src/apis/discovery/v1';
            ~~~~~~~~~


src/auth.ts:12:10 - error TS2300: Duplicate identifier 'Discovery'.

12 import { Discovery } from 'googleapis/build/src/apis/discovery/v1';
            ~~~~~~~~~

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @google/clasp@1.5.3 build: `tsc --project tsconfig.json && npm i -g --l
oglevel=error`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @google/clasp@1.5.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\GL000017\AppData\Roaming\npm-cache\_logs\2018-08-23T18_48_
39_488Z-debug.log

## Steps to Reproduce the Problem

  1. Clone the project
  1. Run the steps under "Develop" section
  1. See the error

## Specifications

  - Node version (`node -v`): v8.11.4
  - Version (`clasp -v`): 1.5.3
  - OS (Mac/Linux/Windows): Windows 7

**Comment by valen22br** [comment]
I'm getting error 403 when I try to push my branch. Do you know how to solve that?

**Comment by grant** [comment]
Can you press the fork button on the top right, clone that repo, then push to that repo?
After you push, you'll see there is a button to submit a Pull Request.
https://help.github.com/articles/creating-a-pull-request/
https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by grant** [comment]
Fixed with #298.

**Comment by grant** [comment]
I'm going to delete the comments in the other thread since they are off topic. Here they are:

> Hi @grant ,
> I was recreating my development environment and now I can't build clasp anymore using the command npm run build or npm run build-fresh.
> 
> Following is the error:
> 
> tsc --project tsconfig.json && npm i -g --loglevel=error
> 
> src/auth.ts:1:13 - error TS2300: Duplicate identifier 'fs'.
> 
> 1 import * as fs from 'fs';
> ~~
> 
> src/auth.ts:2:13 - error TS2300: Duplicate identifier 'fs'.
> 
> 2 import * as fs from 'fs';
> ~~
> 
> src/auth.ts:11:10 - error TS2300: Duplicate identifier 'Discovery'.
> 
> 11 import { Discovery } from 'googleapis/build/src/apis/discovery/v1';
> ~~~~~~~~~
> 
> src/auth.ts:12:10 - error TS2300: Duplicate identifier 'Discovery'.
> 
> 12 import { Discovery } from 'googleapis/build/src/apis/discovery/v1';
> ~~~~~~~~~
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 2
> npm ERR! @google/clasp@1.5.3 build: tsc --project tsconfig.json && npm i -g --l oglevel=error
> npm ERR! Exit status 2
> npm ERR!
> npm ERR! Failed at the @google/clasp@1.5.3 build script.
> npm ERR! This is probably not a problem with npm. There is likely additional log
> ging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR! C:\Users\GL000017\AppData\Roaming\npm-cache_logs\2018-08-23T18_37_
> 47_980Z-debug.log
> 
>  @valen22br
>  Contributor
> valen22br (Luis Gustavo Grubert Valensuela) 9 hours ago 
> I'm looking at your code, specifically at auth.ts file, and there are some duplicated lines on it.
> I removed the duplicated lines, and it is building successfully. Why were those lines duplicated?
> 
>  @grant
>  Member
> grant 9 hours ago 
> @valen22br Ah, that looks like a separate issue (from a separate change). Can we discuss this in a separate GitHub Issue?
> 
>  @valen22br
>  Contributor
> valen22br (Luis Gustavo Grubert Valensuela) 9 hours ago 
> Yes, no problem!
